modules/core/src/core/tasks/commands/tasks.ts

```diff
@@ -262,7 +262,6 @@ function singleTaskToSpec(
 
 	let fn: PromiseFn | undefined;
 	if (cmd === '$0') {
-		logger.info([cmd, ...args]);
```

Comment:
do we not want this output anymore or is it logged elsewhere in the new impl?

Reply:
I added that while testing and forgot to remove it